src/WebhookResponse/WebhookResponse.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Spatie\WebhookClient\WebhookResponse;
+
+use Illuminate\Http\Request;
+use Spatie\WebhookClient\WebhookConfig;
+
+interface WebhookResponse
+{
+    public function respondToValidWebhookRequest(Request $request, WebhookConfig $config);
```

Comment:
Let's add `Symfony\Component\HttpFoundation\Response` as the return type. Also add this return type to all approriate functions that return the response.